OpenRA.Mods.Common/Widgets/Logic/Ingame/ObserverShroudSelectorLogic.cs

```diff
@@ -210,10 +210,12 @@ public bool HandleKeyPress(KeyInput e)
 				if (e.Key >= Keycode.NUMBER_0 && e.Key <= Keycode.NUMBER_9)
 				{
 					var key = (int)e.Key - (int)Keycode.NUMBER_0;
-					var team = teams.Where(t => t.Key == key).SelectMany(s => s).ToList();
-					if (team.Count == 0)
+					var teamGroup = teams.SingleOrDefault(t => t.Key == key);
```

Comment:
does it need to crash on 0 here?

Reply:
I certainly messed up this condition, corrected.